build.gradle

```diff
@@ -0,0 +1,120 @@
+import java.nio.file.Files
+import java.nio.file.Paths
+
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING
+
+apply plugin: 'java'
+apply plugin: 'idea'
+apply plugin: 'maven-publish'
+
+group "org.logstash.outputs"
+version Files.readAllLines(Paths.get("version")).first()
+
+repositories {
+    mavenCentral()
+}
+dependencies {
+    implementation 'com.azure:azure-core-http-netty:1.13.5'
```

Comment:
We need to explicitly declare all of this deps?

Reply:
Unfortunately yes! We can either get the dependencies from the effective pom or from the previous ruby file created during  ruby install that was done previously